src/app/Helpers/ValidateCreditCardHelper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Helpers;
+
+class ValidateCreditCardHelper
+{
+    public static function validCreditCard ($number) : bool
+    {
+        // Set the string length and parity
+        $number_length=strlen($number);
```

Comment:
Please, put a space between the equal sign. Add a space between the variables and the operators. on the other places on this PR.

Reply:
Modifications made. Thank you !!!